packages/x-components/src/x-modules/search-box/components/search-input-placeholder.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <component
+    :is="animation"
+    v-if="isVisible"
+    class="x-input-placeholder x-search-input-placeholder"
+    mode="out-in"
+  >
+    <span v-if="isBeingAnimated" :key="animationMessage" data-test="search-input-placeholder">
+      {{ animationMessage }}
+    </span>
+    <span v-else-if="$slots.default" data-test="search-input-placeholder">
```

Comment:
Do we have to render a span by default? I'm not sure if this should be defined in the slot.

```
  <template v-else-if="$slots.default">
      <!--
        @slot The default placeholder when the `messages` animation is not active.
      -->
      <slot data-test="search-input-placeholder"></slot>
    </template>
```